### ISSUE TITLE
The `watch-demo` task doesn't work any more

### ISSUE THREAD

**Issue by jelbourn** [angular/material#6396]
If you run `gulp server` and then, say,
```
gulp watch-demo -c autocomplete
```
The server comes up, but none of the demos actually work. The problem appears to be a malformed request:
```
http://localhost:8080/dist/demos/autocomplete/demoBasicUsage/'%20+((__t%20=%20(%20file.name%20))%20==%20null%20?%20%27%27%20:%20__t)%20+%27
```

**Comment by DevVersion** [comment]
I created a fix for that @jelbourn #6406

**Comment by naomiblack** [comment]
thank you @DevVersion!
